src/components/sidebar.tsx

```diff
@@ -0,0 +1,12 @@
+export function Sidebar() {
+  return (
+    <div className="sidebar">
+      <p className="resource">People</p>
+      <p className="resource">Films</p>
+      <p className="resource">Spaceships</p>
+      <p className="resource">Vehicles</p>
+      <p className="resource">Species</p>
+      <p className="resource">Planets</p>
+    </div>
```

Comment:
Remember that the way something looks is distinctly different from the way it behaves, especially when it comes to accessibility and the desired _function_ of the element. Because these are intended to be nav links, I would strongly recommend updating them to be anchor tags, even if they don't go anywhere. We can also use the semantic `<nav>` element to add additional semantic value.